### DISCUSSION TITLE
Pump S100101, after a while, the web stops working.

### DISCUSSION THREAD

**Discussion by franpalacio** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#469] [Q&A]
Hello, I have a S100101 pump with the Tahiti model, and the latest version of firmware v4.0.
First say that I am testing, I have the pump disassembled from the spa, so I am only momentarily testing if the commands and the web interface work.
When I connect the pump everything works correctly, but after a while (I leave it connected and I'm going to do other things), I can't connect to the web interface, the buttons continue to work.
I have a Wi-Fi network with many devices but it shouldn't be the problem, I have other ESP32s with FreeDS (solar surplus manager) and they don't give me any problems.
Any idea what might be going on or how to fix it.
All the best

**Comment by visualapproach** [comment]
Latest version is now 4.1.0
Try updating to this.
Check RSSI if you have bad Wi-Fi at the device
Edit: also, don't keep several browser pages open all the time. Every connection takes memory, which is a bottleneck on the 8266

**Comment by jenswalit** [comment]
Hello, I'm back too. I now also have the 6Wire S100101. my question is which pump should I select in the current version on the WebGui?

Today I wanted to build and test the circuit board in the pump.
Thanks very much

**Reply by visualapproach** [reply]
I think you got it right :-)
